parsec/_parsec_pyi/organization.pyi

```diff
@@ -48,3 +49,5 @@ class OrganizationConfig:
     def sequester_services(self) -> tuple[bytes, ...] | None: ...
     @property
     def is_sequestered_organization(self) -> bool: ...
+    @property
+    def minimum_archiving_period(self) -> int: ...
```

Comment:
Add a doctstring to specify this is in seconds ?
(the other solution is to rename to something like `minimum_archiving_period_secs`)